main.py

```diff
@@ -105,6 +112,19 @@ def plot_img(self, idx):
     def plot_hist(self, idx):
         plot_hist_array(self[idx])
 
+
+    def plot_modified_image(self, idx):
+        delta_image = self[idx]
+        delta_image[delta_image != 0]*=self.enlarge_factor
+        plot_image_array(delta_image)
+
+    def printImageArray(self,idx):
+        delta_image = self[idx]
+        delta_image[delta_image != 0]*=self.enlarge_factor
```

Comment:
This is low priority for now, but please check the coding formats e.g. leave spaces between operators and variables